packages/core/base/src/config.ts

```diff
@@ -1,4 +1,4 @@
-export const AUTH_DOMAIN = "desolate.space";
+export const AUTH_DOMAIN = process.env.SOLANA_AUTH_DOMAIN || "";
```

Comment:
You got to change to process.env.AUTH_DOMAIN too

Reply:
Thanks for catching that.